Library/Formula/devil.rb

```diff
@@ -1,11 +1,53 @@
 class Devil < Formula
   desc "Cross-platform image library"
   homepage "https://sourceforge.net/projects/openil/"
-  url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/DevIL-1.7.8.tar.gz"
-  sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
   revision 1
 
-  bottle :disable, "Can't generate bottles until builds with either Clang or GCC-5"
+  stable do
+    url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/" \
+      "DevIL-1.7.8.tar.gz"
+
+    sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
+
+    # fix compilation issue for iluc.c
```

Comment:
Should this be "`iluc.c`" or "`ilur.c`"? Wasn't introduced by this change, but may as well fix it while working near the patch code.